src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/LSFRestJobManager.groovy

```diff
@@ -187,6 +187,19 @@ class LSFRestJobManager extends BatchEuphoriaJobManagerAdapter {
         }
 
         requestBody << paramArea
+        if (job.toolScript) {
+            requestBody << "--${headBoundary}\r\n"
+
+            requestBody << [
+                    "Content-Disposition: form-data; name='f1'",
+                    "Content-Type: application/octet-stream",
+                    "Content-Transfer-Encoding: UTF-8",
+                    "Content-ID: <${job.jobName}>",
+                    "",
+                    "${job.toolScript}\r\n",
+            ].join("\r\n")
```

Comment:
Don't want to be a nitpicker but would it make sense to extract \r\n to a custom constant? (NEWLINE / REST_NEWLINE). Is this always the same for different types of servers?

Reply:
Fixed.
HTTP always uses \r\n.